static/icons/caret-down.svg

```diff
@@ -1,3 +0,0 @@
-<svg viewBox="0 0 120 120" transform="rotate(180)" xmlns="http://www.w3.org/2000/svg" id='caret-down'>
```

Comment:
why are these icons deleted

Reply:
I had created rotated icons before I realized I could just pass the code to rotate them (ex: `transform="rotate(180)"`) the same way we're passing down the colors and sizes

Reply:
now instead of `caret-down` `caret-right` `caret-left` `caret-up` there is just `caret` that takes an optional `direction` prop